### DISCUSSION TITLE
Plugin directory

### DISCUSSION THREAD

**Discussion by simonweil** [terraform-linters/tflint#1498] [Ideas]
Does there exist any tflint plugin directory?

**Comment by wata727** [comment]
What do you mean by plugin directory? Something like https://registry.terraform.io/browse/providers ?

**Reply by simonweil** [reply]
Yes, though it could also be just a wiki page updated by the community

**Reply by wata727** [reply]
[GitHub Topics](https://github.com/topics/tflint-ruleset) may be a good alternative.
